{
	"id": "CVE-2023-52323",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2023-52323",
				"description": "PyCryptodome and pycryptodomex before 3.19.1 allow side-channel leakage for OAEP decryption, exploitable for a Manger attack.",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "nvd@nist.gov",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:N/A:N",
							"base_score": 5.9,
							"base_severity": "MEDIUM",
							"temporal_score": 5.9,
							"temporal_severity": "MEDIUM",
							"environmental_score": 5.9,
							"environmental_severity": "MEDIUM"
						}
					}
				],
				"cwe": [
					{
						"source": "nvd@nist.gov",
						"cwe": [
							"CWE-203"
						]
					}
				],
				"references": [
					{
						"source": "cve@mitre.org",
						"url": "https://github.com/Legrandin/pycryptodome/blob/master/Changelog.rst"
					},
					{
						"source": "cve@mitre.org",
						"url": "https://pypi.org/project/pycryptodomex/#history"
					},
					{
						"source": "nvd.nist.gov",
						"url": "https://nvd.nist.gov/vuln/detail/CVE-2023-52323"
					}
				],
				"published": "2024-01-05T04:15:07.763Z",
				"modified": "2024-01-11T17:06:00.26Z"
			},
			"ecosystems": [
				"cpe"
			]
		}
	],
	"detection": [
		{
			"ecosystem": "cpe",
			"criteria": {
				"operator": "OR",
				"criterias": [
					{
						"operator": "OR",
						"criterias": [
							{
								"operator": "OR",
								"criterias": [
									{
										"operator": "OR",
										"criterions": [
											{
												"vulnerable": true,
												"fix_status": {
													"class": "unknown"
												},
												"package": {
													"cpe": "cpe:2.3:a:pycryptodome:pycryptodome:*:*:*:*:*:python:*:*"
												},
												"affected": {
													"type": "semver",
													"range": [
														{
															"lt": "3.19.1"
														}
													]
												}
											}
										]
									},
									{
										"operator": "OR",
										"criterions": [
											{
												"vulnerable": true,
												"fix_status": {
													"class": "unknown"
												},
												"package": {
													"cpe": "cpe:2.3:a:pycryptodome:pycryptodomex:*:*:*:*:*:python:*:*"
												},
												"affected": {
													"type": "semver",
													"range": [
														{
															"lt": "3.19.1"
														}
													]
												}
											}
										]
									}
								]
							}
						]
					}
				]
			}
		}
	],
	"data_source": {
		"id": "nvd-api-cve",
		"raws": [
			"vuls-data-raw-nvd-api-cve/2023/CVE-2023-52323.json"
		]
	}
}
